src/transformers/cache_utils.py

```diff
@@ -781,6 +781,12 @@ def __init__(self, cache_config: CacheConfig) -> None:
         super().__init__(cache_config)
 
         if is_optimum_quanto_available():
+            optimum_quanto_version = version.parse(importlib.metadata.version("optimum-quanto"))
+            print(optimum_quanto_version)
```

Comment:
I think you can remove that print statement.

Reply:
oh yes my bad